### DISCUSSION TITLE
MAC2 output of bdg file contain only called peaks/reads information or all mapped reads?

### DISCUSSION THREAD

**Discussion by TingtingSsl2** [macs3-project/MACS#641] [General]
Hi,

I'm wondering if the MACS2 output of bdg file contains only called peaks/reads information or all mapped reads?

Thank you for your help!

**Comment by taoliu** [comment]
Hi @TingtingSsl2, It was generated by considering all the reads after filtering (by default, filtering the duplicated reads, unmapped reads, unmapped pairs, etc.). You can read [this document ](https://macs3-project.github.io/MACS/docs/Advanced_Step-by-step_Peak_Calling.html)to get some idea of how the bdg files can be generated step by step using different MACS subcommands (this document simulates the process of `callpeak` by using other subcommands).

**Reply by TingtingSsl2** [reply]
Hi @taoliu,

Thank you for answering my question and for providing the updated MACS3 with more handy tools. I have a quick followup question: does the BED file generated by MACS2 peak calling only include significant peaks (as defined during peak calling)? If so, does the BDG file contain all reads after filtering, while the BED file contains only the significant peaks?

Thank you again for your help.
